pkg/provider/aws/resources/rds.go

```diff
@@ -107,6 +107,85 @@ type (
 	}
 )
 
+type RdsInstanceCreateParams struct {
+	AppName string
+	Refs    []core.AnnotationKey
+	Name    string
+}
+
+// Create takes in an all necessary parameters to generate the RdsInstance name and ensure that the RdsInstance is correlated to the constructs which required its creation.
```

Comment:
I'm not a huge fan of the copy/paste preamble. Feels like it should go somewhere common (eg a package-level doc or an interface).

Reply:
ok i can delete these for now